tests/fixtures/favicon.html

```diff
@@ -6,9 +6,9 @@
 <link rel="icon" href="favicon.png">
 
 
+
```

Comment:
what happened to all the changed whitespace here? Did something in the parsing change?